.circleci/config.yml

```diff
@@ -123,6 +123,17 @@ jobs:
       - run: echo 're-enable tests here'
       - run: echo 'make utils.test'
 
+  check-vulnerabilities:
+    <<: *python_node_postgres_images
+    steps:
+      - *attach_workspace
+      - run: make admin.setup-dev
+      - run: make admin.check-vulnerabilities
+      - run: make api.setup-dev
+      - run: make api.check-vulnerabilities
+      - run: make utils.setup-dev
+      - run: make utils.check-vulnerabilities
```

Comment:
I guess with this section, would it be better do it like this:

```
- run: make setup-dev
- run: make check-vulnerabilities
```

?

Meaning that if we add checks to the status project (and others we add), this will run those too.

Reply:
Changed